api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,22 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Global target percentage of requests that will be force traced if the
+    // x-client-trace-id header is set. Must be an integer number between 0 and
+    // 100.
+    uint32 client_enabled = 3 [(validate.rules).uint32.lte = 100];
+
+    // Global target percentage of requests that will be traced after all other
+    // checks have been applied (force tracing, sampling, etc.). Must be a
+    // number between 0 and
+    // 100.
+    uint32 global_enabled = 4 [(validate.rules).uint32.lte = 100];
+
+    // Global target percentage of requests that will be randomly traced.
+    // Specified as ten-thousandths of a percent (i.e., in 0.01% increments),
+    // using integer numbers in the range 0-10000.
```

Comment:
This is a problem in runtime as well, but I can't help but feel that we should have a standard practice for specifying percentages in the Envoy proto API. In some places, we use `double`, other places `uint32`, with different degrees of granularity. Ideally we would have a `Percent` message type in the API that would also capture the `lte` constraint.

@mattklein123 @wora for comment and possible addition to STYLE.md. I don't think there is anything actionable for this PR @hausdorff.

Reply:
+1. I'm sad we didn't do this in the original v2 conversion. Oh well. Might as well start now. Can we just a common Percent message per @htuch? I would probably just use a double and clamp it between 0.0 and 1.0. Internally, we can convert to integer as needed (e.g., a number out of 10,000 or 100,000 or whatever).

Reply:
I didn't want to cause waves by having the API not match up with the implementation, but I'd also prefer the wrapping a `double` in a `Percent` message type. So let's do that.

Fixed.